src/components/Categories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
+}
```

Comment:
- [ ] The categories should display only a button with the text "Check status". Please implement it